### PR TITLE
code challenge 16

### PR DESCRIPTION
**Pull request by mparkersimms**
[ x] Top-level README “Table of Contents” is updated
[x ] Feature tasks for this challenge are completed
[ x] Unit tests written and passing
[x ] “Happy Path” - Expected outcome
[ x] Expected failure
[ x] Edge Case (if applicable/obvious)
[ x] README for this challenge is complete
[ x] Summary, Description, Approach & Efficiency, Solution
[ x] Link to code
[x ] Picture of whiteboard

### CODE REVIEW

File: utilities/app/src/test/java/utilities/Tree/TreeTester.java [hunk 1]
```diff
@@ -82,9 +82,20 @@
         System.out.println(binaryTree.contains("d"));
         System.out.println(binaryTree.contains("e"));
 
-
-
-
+    }
+    @Test public void maxValueTest() throws Exception {
+        Tree testTree = new Tree();
+        testTree.root = new Node(1, null, null);
```
Comment (Mattpet26) [inline utilities/app/src/test/java/utilities/Tree/TreeTester.java:88]: Why does your node accept three values?
